### PR TITLE
globs files defined in conf_files=

### PR DESCRIPTION
**Pull request by Gottox**
Depends on https://github.com/voidlinux/void-packages/pull/2934

### CODE REVIEW

**Comment by Vaelatern** [comment]
Doesn't work.
Tested with xinetd.
When I tried with `{}` globbing, `=> ERROR: xinetd-2.3.15_6: '/etc/xinetd.{conf,d/serv{ers,ices}}' configuration file not in PKGDESTDIR!`
When I tried with `/etc/xinetd.*` I say a notice, but not warning or error, `/void-packages/common/hooks/pre-pkg/99-pkglint.sh: line 29: [: /destdir//xinetd-2.3.15//etc/xinetd.conf: binary operator expected`

**Comment by Gottox** [comment]
Please recheck. brace expansion is not supported.

File: common/hooks/pre-pkg/99-pkglint.sh [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # This hook checks for common issues related to void.
 
+source ${XBPS_COMMONDIR}/environment/pkg/extglob.sh
```
Comment (xtraeme) [inline common/hooks/pre-pkg/99-pkglint.sh]: Hmm those both two lines sourcing the files shouldn't be required.

Comment (Gottox) [inline common/hooks/pre-pkg/99-pkglint.sh]: You're right for ``00-gen-pkg``. But ``99-pkglint`` does not source this file.

**Comment by Vaelatern** [comment]
Checked with lightdm this time.

    conf_files= "
         /etc/lightdm/*.conf                                                     
         /etc/pam.d/lightdm*"                                                    

and the necessary files were created, listed below.

    conf_files="
	    /etc/lightdm/keys.conf
	    /etc/lightdm/lightdm.conf
	    /etc/lightdm/users.conf
	    /etc/pam.d/lightdm
	    /etc/pam.d/lightdm-greeter
	    /etc/pam.d/lightdm-autologin"

I can't do `conf_files="/etc/**/*"`, the regex needs to be a little precise about files.

**Comment by Gottox** [comment]
All this bash madness is not supported at the moment. Just plain simple * and ? expansions.

Let's focus on that and extend it later if needed.

**Comment by Gottox** [comment]
Any further comments?

**Pull request merged**